determined-ai/determined
# fix: prevent checkpoint modals from closing on their own [ET-116] [ET-120]

## Description
Addresses the bugs from release part here https://github.com/determined-ai/release-party-issues/issues/1594




## Test Plan

- Go to the checkpoints table of a single trial experiment
- Click on the flag in the Checkpoint column to open the checkpoint modal
- Switch between browser tabs and confirm that the modal doesn't close
- Test registering checkpoint
- Repeat all tests on the Workloads table on the Overview tab of the same experiment (below metrics graph)
- Repeat all tests by clicking the "Best Checkpoint" card on the Overview tab
- Repeat all tests on the Trials table of a multi trial experiment





## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
